lib/rules/no-descending-specificity/index.js

```diff
@@ -55,9 +56,11 @@ const rule = function(actual) {
         const index = rule.selector.indexOf(trimSelector);
         // Resolve any nested selectors before checking
         resolvedNestedSelector(selector, rule).forEach(resolvedSelector => {
-          parseSelector(resolvedSelector, result, rule, s =>
-            checkSelector(s, rule, index, comparisonContext)
-          );
+          parseSelector(resolvedSelector, result, rule, s => {
+            if (isStandardSyntaxSelector(resolvedSelector)) {
```

Comment:
The convention, in this file and others, is to return early when using the `isStandard*` utils. We should probably follow it here too:

```js
parseSelector(resolvedSelector, result, rule, s => {
  if (isStandardSyntaxSelector(resolvedSelector)) {
    return;
  }
  checkSelector(s, rule, index, comparisonContext);
}
```